deploy/clowdapp.yaml

```diff
@@ -22,21 +22,6 @@ objects:
           enabled: true
       podSpec:
         image: ${IMAGE}:${IMAGE_TAG_MANAGER}
-        initContainers:
-          - command:
-            - /manifest/push.sh
-            - RedHatInsights/manifests
-            - ${MANIFEST_PUSH_BRANCH}
```

Comment:
Please remove MANIFEST_PUSH_BRANCH from all places it is used.

Reply:
We can also remove `github-patchman-bot` secret as it's used just to push manifest to github repo.

Reply:
removed `MANIFEST_PUSH_BRANCH` and `github-patchman-bot`